### PR TITLE
third-party-repo org_id transition

### PR DESCRIPTION
**Pull request by lahavyuv86**
# Description

Updating db queries with new org_id for third-party-repo service.
Jira ticket: https://issues.redhat.com/browse/THEEDGE-1984

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [x] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
4d4c170b-f676-42a6-abd3-3b3d114d8f06

**Comment by ldjebran** [comment]
we introduced new functions yesterday take a look how we can do it https://github.com/RedHatInsights/edge-api/pull/1154
Please update your PR accordingly.

File: pkg/routes/thirdpartyrepo.go [hunk 1]
```diff
@@ -77,14 +77,14 @@ func CreateThirdPartyRepo(w http.ResponseWriter, r *http.Request) {
 	}
 	ctxServices.Log.Info("Creating custom repository")
 
-	account, err := common.GetAccount(r)
+	account, orgID, err := common.GetAccountOrOrgIDFromContext(r.Context())
```
Comment (ldjebran) [inline pkg/routes/thirdpartyrepo.go:80]: in routes modules we use:
```go
account, orgID := readAccountOrOrgID(w, r, contextServices.Log)
if account == "" && orgID == "" {
    // logs and response handled by readAccountOrOrgID
    return
}
```

File: pkg/routes/thirdpartyrepo.go [hunk 2]
```diff
@@ -125,13 +125,13 @@ func GetAllThirdPartyRepo(w http.ResponseWriter, r *http.Request) {
 	var tprepo []models.ThirdPartyRepo
 	var count int64
 
-	account, err := common.GetAccount(r)
+	account, orgID, err := common.GetAccountOrOrgIDFromContext(r.Context())
```
Comment (ldjebran) [inline pkg/routes/thirdpartyrepo.go:128]: in routes we use 

```go

account, orgID := readAccountOrOrgID(w, r, contextServices.Log)
if account == "" && orgID == "" {
    // logs and response handled by readAccountOrOrgID
    return
}
```

File: pkg/routes/thirdpartyrepo.go [hunk 3]
```diff
@@ -187,12 +187,13 @@ func ThirdPartyRepoCtx(next http.Handler) http.Handler {
 				respondWithAPIError(w, ctxServices.Log, responseErr)
 				return
 			}
-			account, err := common.GetAccount(r)
-			if err != nil || tprepo.Account != account {
+			account, orgID, err := common.GetAccountOrOrgIDFromContext(r.Context())
```
Comment (ldjebran) [inline pkg/routes/thirdpartyrepo.go:190]: in routes we use 

```go

account, orgID := readAccountOrOrgID(w, r, contextServices.Log)
if account == "" && orgID == "" {
    // logs and response handled by readAccountOrOrgID
    return
}

File: pkg/routes/thirdpartyrepo.go (hunk 1 above)
Comment (lahavyuv86) [reply pkg/routes/thirdpartyrepo.go:80]: fixed

File: pkg/routes/thirdpartyrepo.go (hunk 2 above)
Comment (lahavyuv86) [reply pkg/routes/thirdpartyrepo.go:128]: fixed

File: pkg/routes/thirdpartyrepo.go (hunk 3 above)
Comment (lahavyuv86) [reply pkg/routes/thirdpartyrepo.go:190]: fixed

**Pull request merged**